support/cas-server-support-webauthn-core/src/test/java/com/yubico/core/WebSessionWebAuthnCacheTests.java

```diff
@@ -0,0 +1,70 @@
+package com.yubico.core;
+
+import com.yubico.webauthn.data.*;
+import lombok.val;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.springframework.mock.web.MockHttpServletRequest;
+import org.springframework.mock.web.MockHttpSession;
+import org.springframework.web.context.request.RequestContextHolder;
+import org.springframework.web.context.request.ServletRequestAttributes;
+
+import java.nio.charset.StandardCharsets;
+
+import static org.junit.Assert.assertNull;
+import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
These seem slightly suspicions: You should only be pulling in `import static org.junit.jupiter.api.Assertions.*;`